apparmor.d/abstractions/lxqt

```diff
@@ -0,0 +1,30 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Besanon <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+  include <abstractions/fonts>
+  include <abstractions/freedesktop.org>
+  include <abstractions/qt5>
+  include <abstractions/mesa>
```

Comment:
Remove `include <abstractions/mesa>` it is part of `include <abstractions/graphics>`